File: C_Sharp_Course/C_Sharp_Course/Files/numbers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace C_Sharp_Course
{
    class WorkingWithNumbers
    {
        static void Numbers(string[] args)
        {
            int num = 6;

            Console.WriteLine(55);
            Console.WriteLine(-5.151);

            Console.WriteLine(5 + 5);
            Console.WriteLine(5 * 5);
            Console.WriteLine(6 - 5);
            Console.WriteLine(10 / 5);
            Console.WriteLine(7 % 5);
            Console.WriteLine(4 + 3 * 2);
            Console.WriteLine((4 + 3) * 2);

            Console.WriteLine(5.0 + 8.1);
            Console.WriteLine(5 + 8.1);

            Console.WriteLine(num);
            num++;
            Console.WriteLine(num);
            num--;
            Console.WriteLine(num);

            Console.WriteLine(Math.Abs(-5));
            Console.WriteLine(Math.Pow(3, 2));
            Console.WriteLine(Math.Pow(3.8, 2));
            Console.WriteLine(Math.Sqrt(36));
            Console.WriteLine(Math.Max(4, 90));
            Console.WriteLine(Math.Min(4, 90));
            Console.WriteLine(Math.Round(4.3));
            Console.WriteLine(Math.Round(4.6));

            Console.ReadLine();
        }
    }
}

File: Java_Course/src/App/Numbers.java
package App;

public class Numbers {
	public static void numbers(String [] args) {
		
		System.out.println( 3 + 4 );
		System.out.println( 3 - 2 );
		System.out.println( 3 * 4 );
		System.out.println( 3 / 4 );		//dividing two integers returns also an integer

		System.out.println();
		
		System.out.println( 3.0 + 4.0 );
		
		System.out.println();
		
		System.out.println( 10 % 3 );
		
		System.out.println();
		
		int myInt = 3;
		double myDouble = 5.5;
		
		System.out.println( myInt + 5 );
		System.out.println( myInt + myDouble );
		
		System.out.println();
		
		double myNegDouble = -5.5;

		System.out.println( Math.abs(myNegDouble) );
		System.out.println( Math.pow(myInt, 3) );
		System.out.println( Math.sqrt(36) );
		System.out.println( Math.min(2, 6) );
		System.out.println( Math.max(2, 6) );
		System.out.println( Math.round(myDouble) );
		System.out.println( Math.random());
		
		System.out.println();
	}
}
